snap/info.go

```diff
@@ -56,6 +56,9 @@ type PlaceInfo interface {
 
 	// CommonDataHomeDir returns the per user data directory common across revisions of the snap.
 	CommonDataHomeDir() string
+
+	// XdgRuntimeDir returns the XDG_RUNTIME_DIR directories of the snap.
```

Comment:
@zyga - this is one directory per user, but cleanup will be a glob since multiple users might have used the snap and have the directory. Ie:

- I launch a snap 'foo' and need XDG_RUNTIME_DIR=/run/user/1000/snap.foo (ie UserXdgRuntimeDir())
- you launch a snap 'foo' and need XDG_RUNTIME_DIR=/run/user/1001/snap.foo (ie UserXdgRuntimeDir())
- the sysadmin does 'snap remove foo' (ie XdgRuntimeDir() glob for /run/user/*/snap.foo

I'll improve the comments and function names.